app/views/apis/detail.html

```diff
@@ -202,7 +202,7 @@
                 </div>
 
                 <iframe iframe-auto-size
-                        local-iframe="{{getTrustedUrl(settings.currentPath + 'swagger-console.html?url=' + api.url + '&host=' + api.swaggerPreferences.host + '&basePath=' + api.swaggerPreferences.basePath)}}">
```

Comment:
@aspear I believe this change was not intended as we are still using the local-iframe to avoid the X-frame options issue.